src/states_screens/race_result_gui.cpp

```diff
@@ -1410,6 +1413,36 @@ void RaceResultGUI::backToLobby()
                     false, false, NULL, true /* ignoreRTL */);
             }
         }
+
+        if (race_manager->getMinorMode() != RaceManager::MINOR_MODE_SOCCER
+            && race_manager->getMinorMode() != RaceManager::MINOR_MODE_EASTER_EGG)
```

Comment:
Here it would be pfereable to use race_manager->modeHasLaps() to determine whether it is appropriate to show this section. then the "9999" laps check could also be removed. This would be preferable because if we add more modes in the futures, we won't need to search for places where lists of modes were hardcoded.